apiary.apib

```diff
@@ -304,32 +304,108 @@ A list of objects containing the following parameters:
 |  Name | Description | Example |
 | ---------------| ------------| --------|
 | session_id | unique ID of the session | fOBg4dXAPTHSoil6LywgfYA |
+| formatted_id | unique formatted ID of the session | 123-456-789 |
 | agent_id | ID of the agent | 2 |
-| viewer_link | the link that must be opened in a ***viewer*** browser | https://session.surfly.com/fHrD6TMmRrurMevgidxyw |
+| viewer_link | the link that must be opened in a ***viewer*** browser | https://surfly.com/123-456-789 |
 | start_time | time the session started | 2016-06-17T14:30:44.026111Z |
 | duration | length of the session (in seconds) | 123 |
-| queue_id |  | 3927 |
```

Comment:
It is not removed in fact. I only removed it from the docs to prevent new users from using it.